src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetAsciiTests.cs

```diff
@@ -83,24 +83,22 @@ public void GetAscii(string unicode, int index, int count, string expected)
             }
             Assert.Equal(expected, new IdnMapping().GetAscii(unicode, index, count));
         }
-        
+
         [Fact]
-        public void TestGetAsciiWithDot()
+        [PlatformSpecific(TestPlatforms.AnyUnix ^ TestPlatforms.OSX)]
+        public void TestGetAsciiWithDot_Succeeds()
```

Comment:
Isn't this ignoring the comment "some versions of Linux succeed and others throw exception"?